.rubocop.yml

```diff
@@ -69,6 +71,16 @@ RSpec/VerifiedDoubles:
     - 'spec/reek/context/method_context_spec.rb'
     - 'spec/reek/context/module_context_spec.rb'
 
+# FIXME: This seems to be a rubocop-rspec bug. The warning says:
+#   spec/reek/report/code_climate/code_climate_report_spec.rb:8:1: C: RSpec/FilePath: Spec path should end with reek/report/code_climate_report*_spec.rb
+#   RSpec.describe Reek::Report::CodeClimateReport do
```

Comment:
I think this means it expects a CodeClimate namespaces to go with the `code_climate` directory.